requirements.txt

```diff
@@ -15,6 +15,7 @@ requests>=2.20.0, <3.0
 s3fs>=0.3.0, <1.0  # Needs to be at least 0.3.0 to make use of `cachable` attribute on S3FileSystem
 SQLAlchemy>=1.2.0, <2.0
 tables>=3.4.4, <3.6  # 3.6.0 breaks on install, HDF5 library not found
+tensorflow>=2.0.0, <=2.1.0  # only V2 supported for saving and loading. V1 requires HDF5 and serializes differently
```

Comment:
Can you put this in `setup.py` under `extras_require` under a `tensorflow` key so users don't need to have `tensorflow` installed when installing `kedro`, but can choose to install it with `pip install "kedro[tensorflow]"` if they want to use a `TensorflowDataSet`.